tensorflow/python/lib/io/file_io.py

```diff
@@ -587,6 +590,41 @@ def is_directory_v2(path):
     return False
 
 
+@tf_export(v1=["gfile.NeedsTempLocation"])
```

Comment:
Not sure adding to the v1 API gets us anything at this point; is there a reason to add it to v1 separately?

Reply:
Might be needed for TF 1.15, but probably will be reject during API review as I don't think we add new APIs under TF1.15

Reply:
For TF API Owners: Let's skip the v1 change.

Reply:
Okay